src/hotspot/cpu/aarch64/continuationFreezeThaw_aarch64.inline.hpp

```diff
@@ -149,10 +155,16 @@ inline void FreezeBase::relativize_interpreted_frame_metadata(const frame& f, co
   // extended_sp is already relativized by TemplateInterpreterGenerator::generate_normal_entry or
   // AbstractInterpreter::layout_activation
 
+  // The interpreter native wrapper code adds space in the stack equal to size_of_parameters()
+  // after the fixed part of the frame. For wait0 this is equal to 3 words (this + long parameter).
```

Comment:
```suggestion
  // after the fixed part of the frame. For wait0 this is equal to 2 words (this + long parameter).
```
Isn't that 2 words, not 3?

Reply:
The timeout parameter is a long which we count as 2 words: https://github.com/openjdk/jdk/blob/0e3fc93dfb14378a848571a6b83282c0c73e690f/src/hotspot/share/runtime/signature.hpp#L347
I don't know why we do that for 64 bits.

Reply:
OK, I think there are historical or technical reasons why it's hard to change, because of the way the JVM spec is written.